src/PersonalSite.Tests/PersonalSite.Domain.API.UnitTests/PersonalSite.WebApi.UnitTests.csproj

```diff
@@ -9,6 +9,9 @@
     </PropertyGroup>
 
     <ItemGroup>
+        <PackageReference Include="Moq" Version="4.16.1" />
+        <PackageReference Include="MyTested.AspNetCore.Mvc" Version="5.0.0" />
```

Comment:
I was thinking to have that tested by using dotnet official functional test library https://www.nuget.org/packages/Microsoft.AspNetCore.Mvc.Testing. 

https://docs.microsoft.com/en-us/aspnet/core/test/integration-tests?view=aspnetcore-6.0

As the other one we can do it in pair next week as well